skel2.ebuild

```diff
@@ -0,0 +1,86 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+PYTHON_COMPAT=( python2_7 python3_5 )
+
+EAPI=6
+
+inherit autotools flag-o-matic python-r1 readme.gentoo-r1 toolchain-funcs
+
+MY_PN=${PN/-/_}
+MY_PV=${PV/_pre/-pre}
+
+DESCRIPTION="The example ebuild"
+HOMEPAGE="https://foo.example.org/"
+SRC_URI="https://foo.example.org/${MY_PN}/${MY_PN}-${MY_PV}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc openmp test"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	openmp? ( dev-libs/boost:= )"
+DEPEND="${RDEPEND}
+	doc? ( app-doc/doxygen )
+	test? ( dev-util/cppunit )"
+
+S=${WORKDIR}/${MY_PN}-${MY_PV}-src
+
+PATCHES=(
+	"${FILESDIR}"/${PN}-1.2.3-build-system.patch
+	"${FILESDIR}"/${PN}-1.2.3-desktop.patch
+)
+DOCS=( "${FILESDIR}"/${PN}-manual.pdf )
+HTML_DOCS=( doc/html/{index.html,style.css,logo.gif} )
+
+pkg_pretend() {
+	[[ ${MERGE_TYPE} != binary ]] && use openmp && tc-check-openmp
+}
+
+pkg_setup() {
+	[[ ${MERGE_TYPE} != binary ]] && use openmp && tc-check-openmp
+}
+
+src_prepare() {
+	default
+	eautoreconf
+}
+
+src_configure() {
+	# codebase uses C++11 features
+	append-cxxflags -std=c++11
+
+	econf \
+		--disable-static \
+		$(use_enable static static-libs) \
+		$(use_enable openmp)
+}
+
+src_install() {
+	default
```

Comment:
Maybe add a `find "${ED}" -name '*.la' -delete` call too, since it's autotools anyway.